### PR TITLE
feature(196): add stubObj

### PR DESCRIPTION
**Pull request by BjornMelgaard**

### CODE REVIEW

File: src/index.d.ts [hunk 1]
```diff
@@ -284,6 +284,11 @@ declare namespace RamdaAdjunct {
          */
         stubNull():  null;
 
+        /**
+         * A function that returns new empty object; different reference for every call.
```
Comment (char0n) [inline src/index.d.ts]: This function returns a new empty object.

File: src/index.d.ts [hunk 2]
```diff
@@ -284,6 +284,11 @@ declare namespace RamdaAdjunct {
...
+         */
+        stubObj():  {};
```
Comment (char0n) [inline src/index.d.ts]: Again shouldn't we infere the type, not the value ?

File: src/stubObj.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import { always } from 'ramda';
+
+/**
+ * A function that returns new empty object; different reference for every call.
```
Comment (char0n) [inline src/stubObj.js]: This method returns a new empty object.

File: src/stubObj.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+ *
+ * @func stubObj
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/2.1.0|v2.1.0}
+ * @category Function
+ * @sig ... -> {}
+ * @return {{}}
```
Comment (char0n) [inline src/stubObj.js]:
```javascript
@return {Object} Returns the new empty object.


```

File: src/stubObj.js [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+ * @example
+ *
+ * RA.stubObj(); //=> {}
+ * RA.stubObj(1, 2, 3); //=> {}
+ */
+const stubObj = always({});
```
Comment (char0n) [inline src/stubObj.js]:
```javascript
const stubObj = () => ({});
```
We want new obj instance on every call

File: test/stubObj.js [hunk 6]
```diff
@@ -0,0 +1,11 @@
+import * as RA from '../src/index';
+import eq from './shared/eq';
+
+describe('stubObj', function() {
+  it('tests `function` that returns `{}`', function() {
```
Comment (char0n) [inline test/stubObj.js]: it('tests `function` that returns new empty object',

File: test/stubObj.js [hunk 7]
```diff
@@ -0,0 +1,11 @@
+import * as RA from '../src/index';
+import eq from './shared/eq';
+
```
Comment (char0n) [inline test/stubObj.js]: extra new line pls

File: test/stubObj.js [hunk 8]
```diff
@@ -0,0 +1,11 @@
...
+describe('stubObj', function() {
+  it('tests `function` that returns `{}`', function() {
+    eq(RA.stubObj(), {});
+    eq(RA.stubObj([1]), {});
+    eq(RA.stubObj(new Array()), {});
+    eq(RA.stubObj(1, 2, 3), {});
+  });
+});
```
Comment (char0n) [inline test/stubObj.js]:
```javascript
context('when called', function() {
  specify('should always return new empty object', function() {
 ....
  });
});

File: src/index.js [hunk 9]
```diff
@@ -60,6 +60,7 @@ export { default as isNotPair } from './isNotPair';
 // Function
 export { default as stubUndefined } from './stubUndefined';
 export { default as stubNull } from './stubNull';
+export { default as stubObj } from './stubObj';
```
Comment (char0n) [inline src/index.js]: lets create alias also
```javascript
export { default as stubObject } from './stubObj';

File: src/stubObj.js [hunk 10]
```diff
@@ -0,0 +1,19 @@
+import { always } from 'ramda';
+
+/**
+ * A function that returns new empty object; different reference for every call.
+ *
+ * @func stubObj
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/2.1.0|v2.1.0}
+ * @category Function
+ * @sig ... -> {}
```
Comment (char0n) [inline src/stubObj.js]: add alias info
```
@aliases stubObject
```

**Comment by char0n** [comment]
@BjornMelgaard can you pls resolve conflicts ?

**Comment by BjornMelgaard** [comment]
@char0n arent they resolved already?

**Comment by char0n** [comment]
Ahh sorry, I am able to squash and merge. Did not notice. Good job on this PR !

**Pull request merged**